assets/helpers/tracking/acquisitions.js

```diff
@@ -2,26 +2,36 @@
 
 // ----- Imports ----- //
 
+
+import * as ophan from 'ophan';
```

Comment:
What happens if ophan gets imported twice on one page? Doesn't it do a bunch of setup stuff on import? Will it do this twice?

Reply:
It only sends one pageview. 
<img width="1213" alt="picture 412" src="https://user-images.githubusercontent.com/825398/31344607-0b223dee-ad0b-11e7-8d4a-735df1595fbf.png">